vm/src/macros.rs

```diff
@@ -188,7 +188,7 @@ macro_rules! py_namespace {
 /// use rustpython_vm::obj::objint::PyInt;
 /// use rustpython_vm::pyobject::PyValue;
 ///
-/// let vm = VirtualMachine::new();
+/// let vm: VirtualMachine = Default::default();
```

Comment:
A lot of these can be `VirtualMachine::default()` instead of `: VirtualMachine = Default::default()`